docs/orchestration/nextflow/nextflow-trouble-shooting.md

```diff
@@ -10,4 +10,48 @@ Very minimal container images such as Alpine do not contain the `glibc` librarie
 ### Suggested Solution(s)
 
  * Modify your image to include or mount these dependencies
- * Use an image (or build from a base) that already contains these such as `ubuntu:latest`
\ No newline at end of file
+ * Use an image (or build from a base) that already contains these such as `ubuntu:latest`
+
+## AWS credentials not working when set in the environment
+### Possible Cause(s)
+You are using temporary federated or IAM role temporary credentials that use the AWS_SESSION_TOKEN in addition to AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY. Nextflow does not look for the AWS_SESSION_TOKEN environment variable as detailed at [nextflow/issues/2839](https://github.com/nextflow-io/nextflow/issues/2839)
+
+### Suggested Solution(s)
```

Comment:
Another solution, which would normally be preferred, would be to run Nextflow on an EC2 or ECS container that has an attached IAM role that provides all required permissions.